src/core/persistence/persistence.service.js

```diff
@@ -35,4 +36,34 @@ export class PersistenceService {
 
 		return model;
 	}
+
+	active(model, settings) {
+		settings = settings || this.model.persistence().settings;
```

Comment:
i think there is an easy way :)
JSON.stringify(this.save(settings)) === JSON.stringify(model)